.github/workflows/typos.toml

```diff
@@ -73,4 +73,10 @@ lst = "lst"
 gam = "gam"
 # Regex search term from contrib/solver/solvers/ipopt
 ond = "ond"
+# We regularly use indx in place of index
+indx = "indx"
```

Comment:
Can we instead standardize on `i`, `idx` or `index`?  `indx` seems an unnecessary subset of "index"...